optimum/habana/transformers/trainer.py

```diff
@@ -117,53 +125,69 @@
 if is_datasets_available():
     import datasets
 
-
 if is_safetensors_available():
     import safetensors.torch
 
-
 if is_peft_available():
     from peft import PeftModel
     from peft.utils import PeftType
 
-
 if is_deepspeed_available():
     from accelerate.utils import DeepSpeedSchedulerWrapper
 
-if is_accelerate_available():
-    from accelerate.utils import (
-        load_fsdp_optimizer,
-        save_fsdp_optimizer,
-    )
+if is_accelerate_available("0.28.0"):
+    from accelerate.utils import DataLoaderConfiguration
```

Comment:
We will have Accelerate >= v0.33.0 (cf. https://github.com/huggingface/optimum-habana/blob/ba914504bd52b989d05a4e11916aa19b3ce176f4/setup.py#L35) so we can remove that